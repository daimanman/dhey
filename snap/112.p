{"C":1,"Duration":683355020,"EndTime":1551409146278901900,"Err":"","FileId":"","InputFileName":"","LonhApi":"sendTest","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409145595547000,"Status":1,"TaskId":112,"Type":"LONH","Url":"","Z":""}