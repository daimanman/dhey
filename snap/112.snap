{"AvgTotal":0.6763755,"Fastest":0.6763755,"Slowest":0.6763755,"Average":0.6763755,"Rps":1.4784686908381512,"AvgConn":0,"AvgDNS":0,"AvgReq":0.0009974,"AvgRes":0,"AvgDelay":0.6693942,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0.0009974,"ReqMax":0.0009974,"ResMin":0.0009974,"ResMax":0,"DelayMax":0.6693942,"DelayMin":0.6693942,"Lats":[0.6763755],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0.0009974],"ResLats":[0],"DelayLats":[0.6693942],"Total":676375500,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":44,"SizeReq":44,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.6763755,"Count":1,"Frequentcy":1},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0},{"Mark":0.6763755,"Count":0,"Frequentcy":0}],"TaskId":112,"Duration":0.6763755}